.eslintrc.json

```diff
@@ -106,9 +106,9 @@
             "never"
         ],
         "patternfly-react/import-tokens-icons": "error",
-        "patternfly-react/no-anonymous-functions": "error",
+        "patternfly-react/no-anonymous-functions": "off",
         "prefer-const": "error",
-        "prettier/prettier": "error",
```

Comment:
Disabled these rules temporarily as changes to linting deps are now causing a lot of errors to be thrown. I've created #8657 and #8658 as followup issues.